### ISSUE TITLE
[wg/wot] Web of Things Working Group rechartering

### ISSUE THREAD

**Issue by ashimura** [w3c/strategy#375]
New charter proposal, reviewers please take note.

# Charter Review

[Draft Charter: Web of Things Working Group](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-draft.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing WG recharter 
* [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F07%2Fwot-wg-2022.html&doc2=https%3A%2F%2Fw3c.github.io%2Fwot-charter-drafts%2Fwot-wg-2023-draft.html), and any issue discussion:

---
Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 
* [Web of Things Community Group](https://www.w3.org/groups/cg/wot)
* [Web of Thing Japanese Community Group](https://www.w3.org/groups/cg/wot-jp)

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
* [GitHub repo for the discussion](https://github.com/w3c/wot-charter-drafts)

Anything else we should think about as we review?

**Comment by himorin** [comment]
No comment nor request from i18n

**Comment by matatk** [comment]
APA is happy with the charter as-is—however, [work has been going on to include onboarding, and the accessibility of onboarding, into the charter](https://github.com/w3c/wot-charter-drafts/pull/77). We'd really love to see that included, and it looks like the changes in that PR have been approved; will it be merged?

**Comment by mmccool** [comment]
The onboarding work and the cited PR is part of detailed planning - these details will not go into the charter.  However, we have incorporated some updates from APA into the detailed planning (still in a PR, but we will be meeting to finalized the week of June 19).   Our intention at the moment is to leave the charter as-is with a high-level summary of the planned work.

**Comment by plehegar** [comment]
(PING is interested in reviewing this charter)

**Comment by himorin** [comment]
from tilt-private 32, as minor comment.
opening/closing dl seems to be missing at "WoT-related W3C Groups" section.

**Comment by ashimura** [comment]
Thanks a lot, @himorin !

I've checked the HTML using the Validator again, and the [draft Charter](https://www.w3.org/2023/08/wot-wg-2023-draft.html) is valid now.

Please see also the [vaidator results](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.w3.org%2F2023%2F08%2Fwot-wg-2023-draft.html).

**Comment by plehegar** [comment]
Announced https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0002.html
